src/demolition_spider.py

```diff
@@ -1,49 +1,89 @@
+from dataclasses import dataclass
+import json
+from typing import Any
 from scrapy import Spider
-from scrapy.http import FormRequest
+from scrapy.http import FormRequest, Response
 from scrapy.utils.response import open_in_browser
 from enum import Enum
 
 
 class PermitType(str, Enum):
-    commercial_wrecking_permit = 'Building/Wrecking/Commercial/NA'
-    residential_wrecking_permit = 'Building/Wrecking/Residential/NA'
+    commercial_wrecking_permit = "Building/Wrecking/Commercial/NA"
+    residential_wrecking_permit = "Building/Wrecking/Residential/NA"
+
+
+@dataclass
+class BuildingServicesSearchResult:
+    # TODO validation (regex?)
+    record_number: str  # e.g. WRK2024R-00138
+    record_details_link: str  # URL to the record details page
+    record_type: PermitType
+    project_name: str | None
+    address: str
+    expiration_date: Any | None  # I haven't found any records w/ this yet
+    short_notes: str | None
+
+    def __str__(self):
+        return f"{self.record_type.value}:\n{self.record_number} - {self.project_name} - {self.address}"
+
 
 class DemolitionSpider(Spider):
-    name = 'DemolitionSpider'
-    start_urls = ['https://aca-prod.accela.com/DAYTON/Cap/CapHome.aspx?module=Building&TabName=Building']
+    name = "DemolitionSpider"
+    start_urls = [
+        "https://aca-prod.accela.com/DAYTON/Cap/CapHome.aspx?module=Building&TabName=Building"
+    ]
+    custom_settings = {
+        "LOG_LEVEL": "INFO"  # Default is DEBUG
+    }
+
     permit_type: PermitType = PermitType.commercial_wrecking_permit
-    start_date: str = '08/01/2023' # strftime('%m/%d/%Y')
-    
+    start_date: str = "08/01/2023"  # strftime('%m/%d/%Y')
+    open_in_browser: bool
+
     # TODO type checking around start_date, it should be a date not a str
-    def __init__(self, permit_type: PermitType = None, start_date: str = None, *args, **kwargs):
+    # TODO the results are actually paginated, so right now we're only grabbing the first page
```

Comment:
Interesting... I expect that most days probably aren't seeing more than 10 records, but we could take a look at  alerting in the email if we see 10 entries, increasing the number of results, or even going to the second page.

I'd probably go with the first option. If there are that many, they are probably going to look regardless.